### PR TITLE
Web track2 release

### PR DESCRIPTION
**Pull request by ArcturusZhang**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by jhendrixMSFT — APPROVED** [review]

File: sdk/web/armweb/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Release History
+
+## v0.1.0 (released)
```
Comment (RickWinter) [inline sdk/web/armweb/CHANGELOG.md:3]: This should be a date of release i.e (2021-08-09)

Comment (ArcturusZhang) [reply sdk/web/armweb/CHANGELOG.md:3]: We can have more discussion on how to write the changelog and compose a document accordingly maybe?
But here we are going to release this now...

**Pull request merged**